### DISCUSSION TITLE
OpenDP/Sensitive Data Discussion at 2021 Dataverse Community Meeting

### DISCUSSION THREAD

**Discussion by djbrooke** [opendp/opendp#249] [Getting Involved] [answered]
Hi everyone,

At the Dataverse Community Meeting earlier this summer, @ekraffmiller and @raprasad presented on the DPCreator and discussed an integration with Dataverse (dataverse.org). Here's a link to the video of their presentation:

https://www.youtube.com/watch?v=q3irpQ4rOyU

The discussion about OpenDP starts around the four-minute mark. In the presentation there are other sensitive data-related topics that you may find interesting, including:

- Julie Goldman(@goldmj), Harvard Countway Library: Preparing for Sensitive Data Use Throughout the Data Lifecycle
- Sebastian Karcher (@adam3smith), Qualitative Data Repository: Sensitive Qualitative Data at QDR: Current Processes and Next Steps
- Caroline Wood (Harvard Medical School): Health Policy Research Data
- Aakash Sharma, Håvard D. Johansen, and Thomas Bye Nilsen (DataverseNO UiT Department of Computer Science): Compliant Sharing of Sensitive Data with Dataverse and Lohpi

 Thanks Ellen and Raman for joining us for the meeting!

**Comment by raprasad** [comment, marked as answer]
Thanks, @djbrooke
